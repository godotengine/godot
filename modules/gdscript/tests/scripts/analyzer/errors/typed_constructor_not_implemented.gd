func test():
	# Error
	# Typed conversion constructor checks if base and the provided type are compatible.
	var convert = float[int]([2.0, 40.0]);
