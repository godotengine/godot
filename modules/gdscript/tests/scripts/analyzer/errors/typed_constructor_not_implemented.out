GDTEST_PARSER_ERROR
Typed container constructor can only be used for Arrays.
